### ISSUE TITLE
**Search bugs**

### ISSUE THREAD

**Issue by Panzimy** [valinet/ExplorerPatcher#3871]
**This isn't a very big problem**
I used the shrink the address bar and the icon is having a problem
![image](https://github.com/user-attachments/assets/34a980c0-54c9-4b4f-87d0-c903e76159c8)
The icon is not OK **!** It's placed down

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

Do you have another antivirus installed?

**Comment by Amrsatrio** [comment]
Yes, known limitation with my address bar shrinking method. No short term plans to fix this yet.

**Comment by milesprower2293** [comment]
For now enable compact view to fix it

**Comment by Panzimy** [comment]
> Please post windows version and build. You can find this by running winver. Please include if you are running x86-64 or Arm. And what language your windows is in. Please post explorer patcher version.
> 
> Please include what settings you changed in explorer patcher properties application. These should be screen shots. If you can, please change it to english first.
> 
> Please include any other software you have installed that modifies the UI of windows.
> 
> Do you have another antivirus installed?

Version 23H2 22631.4317. Processor x64. Win. language Czech. EP version  in video

https://github.com/user-attachments/assets/c649ae73-e3ad-47e5-aeca-7cfd1189ad3e


If i know good no changer software was installed. And I don't have any other antivirus


> @Panzimy For now, enable compact view to fix it

And how to do it

**Comment by milesprower2293** [comment]
open file explorer, click on file, then options then on the new window click the view tab and check the option 'Decrease pace between items (compact view) and click apply

**Comment by Panzimy** [comment]
@milesprower2293 Yeah it's now OK thank you

**Comment by Panzimy** [comment]
@Amrsatrio I sayed that the compact mode made this bug OK

**Comment by Amrsatrio** [comment]
Sorry, did not type it correctly.
